tox.ini

```diff
@@ -4,15 +4,14 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py35, py36, py37-npy{115,116,latest}, docs
+envlist = py35, py36, py37-npy{115,116,latest}, docs
 
 [testenv]
 install_command = pip install --no-binary=numcodecs {opts} {packages}
 setenv =
     PYTHONHASHSEED = 42
     # hooks for coverage exclusions based on Python major version
     py35,py36,py37: PY_MAJOR_VERSION = py3
```

Comment:
Actually this line could go as well, it's only needed for marking code to be excluded from coverage when running tests under py2, which we are not running any more.

Reply:
Done. Thanks @alimanfo!